src/contracts/interfaces/IEigenPod.sol

```diff
@@ -168,4 +164,13 @@ interface IEigenPod {
     /// @notice called by the eigenPodManager to increment the withdrawableRestakedExecutionLayerGwei 
     /// in the pod, to reflect a completion of a queued withdrawal as shares
     function incrementWithdrawableRestakedExecutionLayerGwei(uint256 amountWei) external;
+
+    /// @notice called to deposit ETH into the pod
+    function receiveETH() external payable;
```

Comment:
why is this named `receiveETH` instead of just `receive`? the `receive` function is treated specially, like a fallback!

Reply:
ah forgot about the "receive" function, changed.